pkgs/by-name/pl/platformsh/update.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq common-updater-scripts
+#shellcheck shell=bash
+
+set -eu -o pipefail
+
+version=$(curl -s ${GITHUB_TOKEN:+-u ":$GITHUB_TOKEN"} \
+    curl -s https://api.github.com/repos/platformsh/cli/releases/latest | jq -e -r ".tag_name")
```

Comment:
This looks like a mistake (two curl's) :)

Reply:
Thanks look like my copy paste skills should improve https://github.com/NixOS/nixpkgs/pull/319781